src/components/counter-box/CounterBox.tsx

```diff
@@ -0,0 +1,71 @@
+import {StyleSheet, Theme, useTheme} from '@atb/theme';
+import {View} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import React from 'react';
+import {iconSizes} from '@atb-as/theme';
+import {TextNames} from '@atb/theme/colors';
+
+interface StylingCounter {
+  padding: number;
+  lineHeight: number;
+  type: TextNames;
+}
+export const CounterBox = ({
+  count,
+  size = 'normal',
+  accessibilityLabel,
+}: {
+  count: number;
+  size?: keyof Theme['icon']['size'];
+  accessibilityLabel?: string;
+}) => {
+  const styles = useStyles();
+  const {theme} = useTheme();
+
+  if (count < 1) return null;
+  const isSmall: boolean = size === 'small';
+
+  const smallStyling: StylingCounter = {
+    padding: theme.spacings.xSmall,
+    lineHeight: iconSizes.small,
+    type: 'label__uppercase',
+  };
+
+  const normalStyling: StylingCounter = {
+    padding: theme.spacings.small,
+    lineHeight: iconSizes.normal,
+    type: 'body__primary--bold',
+  };
```

Comment:
I think we should support ´large´ as size here as well, since it is in the typing. If we don't want do to that now, the typing of the field size should probably be:
`size?: Exclude<keyof Theme['icon']['size'], 'large'>`